tests/testthat/test-tokens_select.R

```diff
@@ -226,19 +226,19 @@ test_that("tokens_select works as expected for bigrams selected on char, list of
 
 test_that("tokens_select works correctly with collocations objects", {
     expect_equal(
-        as.list(tokens_select(toks_uni, coll_bi)),
+        as.list(tokens_select(toks_uni, coll_bi$collocation)),
```

Comment:
None of these tests are now testing collocations objects, since by slicing out the `$collocation` you are just returning a character vector. Please restore the test objects by removing `$collocation` so we can check the appropriate behaviour, which is to treat this as equivalent to `phrase(coll_bi$collocation)`.